opentelekomcloud/services/cce/resource_opentelekomcloud_cce_node_v3.go

```diff
@@ -383,8 +383,11 @@ func ResourceCCENodeV3() *schema.Resource {
 			"runtime": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 				ForceNew: true,
```

Comment:
bad decision, will recreate old clusters

Reply:
> bad decision, will recreate old clusters

no its not recreating old nodes.
![image](https://user-images.githubusercontent.com/32225815/235702336-278e5762-07f5-4975-a72c-02d8fbd0abc1.png)